qa/tasks/vstart_runner.py

```diff
@@ -422,7 +422,7 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
 
         return proc
 
-    # XXX: for compatibility keep this method same teuthology.orchestra.remote.sh
+    # XXX: for compatibility keep this method same as in teuthology.orchestra.remote.py
```

Comment:
This is incorrect, you either use `teuthology.orchestra.remote.sh` or `teuthology/orchestra/remote.py`, but in the latter case you need to put reference to teuthology repo since you are in ceph repo.

Reply:
oops, I just misinterpreted `.sh` as file extension. Fixed.